### ISSUE TITLE
Make widgets in dashboard tabbable like other reports within a page 

### ISSUE THREAD

**Issue by mattab** [piwik/piwik#9590]
In order to improve accessibility, we would like to make widgets in dashboard tabbable with the keyboard, so one can browse the dashboard with a keyboard.

Reproduce:

 * Press tab key
 * Go to `Skip to main content`
 * Press enter
 * The focus is now on the dashboard content
 * Got: the first image in the first widget, will be focused
 * Expected: the widget Title should be focused instead. When pressing Tab key again, the next widget should be focused.

**Comment by mattab** [comment]
Actually @tsteur suggested this wasn't maybe a good idea, as it prevents users from tabbing into the widget itself. Reverted the PR FYI @Gabriel-Bowater

**Comment by Gabriel-Bowater** [comment]
Okay
